### PR TITLE
Fixed wrong computations. New values align with the figure mentioned in documentation

### PR DESCRIPTION
**Pull request by vinay-hebb**
Congratulations! You've made it this far! Once merged, the article will appear at https://huggingface.co/blog. Official articles 
require additional reviews. Alternatively, you can write a community article following the process [here](https://huggingface.co/blog-explorers).

## Preparing the Article

You're not quite done yet, though. Please make sure to follow this process (as documented [here](https://github.com/huggingface/blog/tree/main?tab=readme-ov-file#how-to-write-an-article-)):

- [ ] Add an entry to [_blog.yml](https://github.com/huggingface/blog/blob/main/_blog.yml).
- [ ] Add a thumbnail. There are no requirements here, but there is a [template](https://github.com/huggingface/blog?tab=readme-ov-file#how-to-get-a-nice-responsive-thumbnail) if it's helpful.
- [ ] Check you use a short title and blog path.
- [ ] Upload any additional assets (such as images) to the Documentation Images [repo](https://huggingface.co/datasets/huggingface/documentation-images/tree/main/blog). This is to reduce bloat in the GitHub base repo when cloning and pulling. Try to have small images to avoid a slow or expensive user experience.
- [ ] Add metadata (such as authors) to your `md` file. You can also specify `guest` or `org` for the authors.
- [ ] Ensure the publication date is correct.
- [ ] Preview the content. A quick way is to paste the markdown content in https://huggingface.co/new-blog. **Do not click publish**, this is just a way to do an early check.

Here is an example of a complete PR: https://github.com/huggingface/blog/pull/2382

## Getting a Review

Please make sure to get a review from someone on your team or a co-author.
Once this is done and once all the steps above are completed, you should be able to merge.
There is no need for additional reviews if you and your co-authors are happy and meet all of the above.

Feel free to add @pcuenca as a reviewer if you want a final check. Keep in mind he'll be biased toward light reviews
(e.g., check for proper metadata) rather than content reviews unless explicitly asked.

### CODE REVIEW

File: train_memory.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Let's break down this graph into key parts:
 
 3. **Forward Pass (1st Loop)**: Memory increases by 2 GB for the output tensor:
```
Comment (pcuenca) [inline train_memory.md:61]:
```suggestion
3. **Forward Pass (1st Loop)**: Memory increases by 1 GB for the output tensor:  
```

File: train_memory.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ Let's break down this graph into key parts:
 
 1. **Model Creation**: Memory increases by 4 GB, corresponding to the model's size:
```
Comment (pcuenca) [inline train_memory.md:51]:
```suggestion
1. **Model Creation**: Memory increases by 2 GB, corresponding to the model's size:
```

**Review by pcuenca — COMMENTED** [review]
Thank you! Looks good to me with my additional suggestions, cc @qgallouedec for confirmation.

**Comment by qgallouedec** [comment]
Thanks a lot! Over the course of many iterations, I haven't updated all my calculations. My bad!

**Review by qgallouedec — APPROVED** [review]

File: train_memory.md [hunk 3]
```diff
@@ -48,19 +48,19 @@ Let's break down this graph into key parts:
...
+   \\( 10{,}000 \times 50{,}000 \text{ weights} + 50{,}000 \text{ biases in } \texttt{float32 }\text{(4 bytes)} \implies (5 \times 10^8) \times 4 \, \text{bytes} = 2 \, \text{GB}. \\)
 
    This memory (in blue) persists throughout execution.
 
 2. **Input Tensor Creation (1st Loop)**: Memory increases by 200 MB matching the input tensor size:
 
    \\( 5{,}000 \times 10{,}000 \text{ elements in } \texttt{float32 }\text{(4 bytes)} \implies (5 \times 10^7) \times 4 \, \text{bytes} = 0.2 \, \text{GB}.\\)
 
-3. **Forward Pass (1st Loop)**: Memory increases by 2 GB for the output tensor:  
+3. **Forward Pass (1st Loop)**: Memory increases by 1 GB for the output tensor:
```
Comment (qgallouedec) [inline train_memory.md:61]: still 5 9 to fix

**Review by qgallouedec — CHANGES_REQUESTED** [review]

**Review by qgallouedec — APPROVED** [review]

**Review by pcuenca — APPROVED** [review]

**Comment by pcuenca** [comment]
Thank you @qgallouedec and @vinay-hebb 🙌

**Pull request merged**